dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3676,6 +3675,17 @@ LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND
     return 0;
 }
 
+LRESULT CShellBrowser::CreateStatusBar()
+{
+    if (!fStatusBar)
+    {
+        fStatusBar = CreateWindow(STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS |
+                                  SBT_NOBORDERS | SBT_TOOLTIPS, 0, 0, 500, 20, m_hWnd, (HMENU)0xa001,
```

Comment:
Not sure `SBT_NOBORDERS` is a valid statusbar window style. Also, replace `SBT_TOOLTIPS` by `SBARS_TOOLTIPS`.
It might be desirable to add `CCS_BOTTOM` too just to be sure.

Reply:
```SBT_NOBORDERS``` is defined here: https://github.com/wine-mirror/wine/blob/master/include/commctrl.h#L294

I will work on your other suggestions though.

Reply:
Yes of course it is defined, like any other random define out there. This doesn't mean it is a valid status bar window style, though.
For example, `SBT_NOBORDERS` is a style to be used together with the `SB_SETTEXT` message, as explained there:
https://learn.microsoft.com/fr-fr/windows/win32/controls/sb-settext

Reply:
Good call, ```SBT_NOBORDERS``` was a hack to add the size gripper on the status bar. I will replace it with the more appropriately named ```SBARS_SIZEGRIP```.
https://learn.microsoft.com/en-us/windows/win32/controls/status-bar-styles